src/InstrumentationEngine/ProfilerManager.h

```diff
@@ -16,6 +16,19 @@
 
 using namespace ATL;
 
+// The following fixes Error C2280 'std::_Uhash_compare<_Kty,_Hasher,_Keyeq>::_Uhash_compare(const std::_Uhash_compare<_Kty,_Hasher,_Keyeq> &)': attempting to reference a deleted function
+// which occurs when compiling using GUID key for std::unordered_map
+namespace std {
+    template<> struct hash<GUID>
+    {
```

Comment:
Do we have a common we could put this into? It seems useful in general.

Reply:
I can move this to common.lib's stdafx.h

---
In reply to: [399475650](https://github.com/microsoft/CLRInstrumentationEngine/pull/231#discussion_r399475650) [](ancestors = 399475650)